content/terraform-docs-common/docs/cloud-docs/api-docs/queries/index.mdx

```diff
@@ -0,0 +1,411 @@
+        "pending-at": "2025-01-15T10:30:00.000Z",
+        "queued-at": "2025-01-15T10:30:05.000Z",
+        "errored-at": null,
+        "canceled-at": null,
+        "running-at": "2025-01-15T10:30:10.000Z",
+        "finished-at": null
+      },
+      "status": "running",
+      "log-read-url": "https://archivist.terraform.io/v1/object/...",
+    },
+    "relationships": {
+      "workspace": {
+        "data": {
+          "type": "workspaces",
+          "id": "ws-ABC123"
+        }
+      },
+      "configuration-version": {
+        "data": {
+          "type": "configuration-versions",
+          "id": "cv-XYZ789"
+        }
+      },
+      "created-by": {
+        "data": {
+          "type": "users",
+          "id": "user-123"
+        }
+      },
+      "canceled-by": {
+        "data": null
+      }
+    },
+    "links": {
+      "self": "/api/v2/queries/query-ABC123"
+    }
+  }
+}
+```
+
+## Cancel a Query Run
+
+This endpoint cancels a Query Run that is currently pending or running.
+
+`POST /queries/:query_id/actions/cancel`
+
+| Parameter   | Description                          |
+|-------------|--------------------------------------|
+| `:query_id` | The ID of the Query Run to cancel.   |
+
+| Status  | Response                  | Reason                                                                |
+|---------|---------------------------|-----------------------------------------------------------------------|
+| [202][] | Empty body                | Successfully canceled the Query Run.                                  |
+| [409][] | [JSON API error object][] | Failed to transition Query Run to a canceled state.                   |
+
+### Sample Request
+
+```shell
+$ curl\
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  https://app.terraform.io/api/v2/queries/query-ABC123/actions/cancel
+```
+
+## Query Run Attributes
+
+Query Runs have the following attributes:
+
+### Status
+
+The `status` attribute indicates the current state of the Query Run. Possible values are:
+
+- `pending` - The Query Run has been created but not yet queued for execution.
+- `running` - The Query Run is currently executing.
+- `finished` - The Query Run has completed successfully.
+- `canceled` - The Query Run was canceled before completion.
+- `errored` - The Query Run encountered an error during execution.
+
+### Status Timestamps
+
+The `status-timestamps` object contains timestamps for when the Query Run entered each status:
+
+- `pending-at` - When the Query Run was created.
+- `queued-at` - When the Query Run was queued for execution.
+- `running-at` - When the Query Run started executing.
+- `finished-at` - When the Query Run completed successfully.
+- `errored-at` - When the Query Run encountered an error.
+- `canceled-at` - When the Query Run was canceled.
+
+### Log Read URL
+
+The `log-read-url` attribute contains a URL to retrieve the logs for the Query Run. 
```

Comment:
One thing that's missing at the end is a `Available Related Resources` section (see [example](https://github.com/hashicorp/atlas/blob/930e4b7aef88bb1cdb137f5081e87ddaa69961f8/frontend/atlas/tests/integration/components/v2/query-container-test.js#L182)). For queries, the list is: `configuration_version`, `created_by`, `configuration_version.ingress_attributes`